samples/Makefile.conf

```diff
@@ -18,3 +18,11 @@
 ifndef BIN_PATH
 BIN_PATH=../../../bin
 endif
+
+MB_TARGET=--target=armv6m-none-eabi -march=armv6m -mfloat-abi=soft 
```

Comment:
I find the name `MB_TARGET` non-obvious. I suggest either `ARMV6M_TARGET` or `MULTIBIT_TARGET`

Reply:
Agreed, updated to MICROBIT_TARGET to be specific.